src/components/DisplayMetrics.js

```diff
@@ -68,7 +72,15 @@ class DisplayMetrics extends React.Component {
   render() {
     const numberOfClicks = this.state.clicks;
     const numberOfImpresions = this.state.impressions;
-    const selectSuggestions = _.uniqWith(this.state.suggestions, _.isEqual);
+    const uniqueCampaign = _.uniqWith(
+      this.state.suggestionsForCampaign,
+      _.isEqual
+    );
+    const uniqeChannel = _.uniqWith(
+      this.state.suggestionsForChannel,
+      _.isEqual
+    );
+    const selectSuggestions = _.concat(uniqeChannel, uniqueCampaign);
```

Comment:
i would also place this into a utils function, to separate business logic from the view

Reply:
648eb83d51cb96ab87ac8c5d2f4df8a04c2c1336 with this change, I also placed previous concatanation into a utils function.